### DISCUSSION TITLE
mutate will indirectly cause the query to be executed once

### DISCUSSION THREAD

**Discussion by VisZhangrong** [Akryum/vue-apollo#1144] [Q&A]
**Describe the bug**
When I execute a mutate, I find that all queries are also executed。

Minimal example:
```
apollo: {
      me: {
            query: projectGetOne,
            update() {
                console.log("update>>>>>>>>>>>>>")
            },
        },
    },
methods： {
     updatePlugin() {
            console.log("mutate>>>>>>>>>>>>>")
            this.$apollo.mutate({
                  mutation: projectUpdateOnePlugin,
                 variables: params
           }).then(() => {
               console.log("mutate11>>>>>>>>>>>>>")
          })
     }
 }
```
Outcome: （sorry， I cant upload a picture ）
mutate>>>>>>>>>>>>>
update>>>>>>>>>>>>>
mutate11>>>>>>>>>>>>>

**Versions**
vue: 2.6.10
vue-apollo: 3.0.5
apollo-client: 2.6.10

**Comment by Akryum** [comment]
I'm not sure to understand what the issue is here.
If you are executing a mutation and receiving new data about an object, that object will get updated in your app. This is expected behavior.
